src/plugins/custom_integrations/public/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { CustomIntegrationPlugin } from './plugin';
+
+// This exports static code and TypeScript types,
+// as well as, Kibana Platform `plugin()` initializer.
+export function plugin() {
+  return new CustomIntegrationPlugin();
+}
+export { CustomIntegrationsSetup, CustomIntegrationsStart } from './types';
```

Comment:
In light of #112602 I wonder if we should start with exporting a config schema that does not include the `enabled` field?

Reply:
I believe all we need to do is add this to the `custom_integrations/server/index.ts` (just realized I commented on the wrong file)

```
import { schema } from '@kbn/config-schema';

export const config = {
  schema: schema.object({})
}
```